packages/shared/lib/core/error/utils/handleClientError.ts

```diff
@@ -0,0 +1,35 @@
+import { API_ERROR_PARAMETERS, CLIENT_ERROR_REGEXES } from '../constants'
+import { ClientError } from '../enums'
+import { IError } from '../interfaces'
+import { handleError } from './handleError'
+import { handleGenericError } from './handleGenericError'
+
+export function handleClientError(error: IError): void {
```

Comment:
i would generalize this. i think we would do the exact same for not only errors with `type: ClientError` right?